model-optimizer/extensions/front/onnx/resize_ext.py

```diff
@@ -43,5 +43,5 @@ def extract(cls, node: Node):
             ONNXResize11Op.update_node_stat(node, attrs)
         else:
             mode = onnx_attr(node, 'mode', 's', default=b'nearest').decode()
-            UpsampleOp.update_node_stat(node, {'mode': mode})
+            ONNXResize10.update_node_stat(node, {'mode': mode})
```

Comment:
@vgavrilo, @lazarevevgeny, we have so many internal operations for resize in `ops` directory: `ONNXResize10`, `ONNXResize11`, etc. I propose to create just one and split them by some attribute(s). And it is possible trigger front- and middle- transformations based on this attribute.

Reply:
It is very hard to debug such superoperation and supertransformation.